src/sdam/server_selection.ts

```diff
@@ -56,7 +56,7 @@ export function sameServerSelector(description?: ServerDescription): ServerSelec
  * server potentially for a write on a secondary.
  */
 export function secondaryWritableServerSelector(
-  wireVersion?: number,
+  wireVersion?: number | null,
```

Comment:
```suggestion
  wireVersion: number | null,
```

If I'm not mistaken it becomes `number | null`, and therefore it will always be provided (no `undefined` essentially).